src/index.html

```diff
@@ -3,144 +3,144 @@
+      <button class="btn btn__home" type="button">Home</button>
+      <button class="btn btn__places" type="button">Places</button>
+      <button class="btn btn__add flaticon-016-add" type="button"></button>
+    </div>
+
+    <!--add new photo form -->
+    <form id="add-photo-form" class="form form-state--closed">
+      <label for="inputCaption" class="form__label">Caption</label>
+      <input type="text" name="text" id="inputCaption" class="form__input" />
+      <label for="inputDate" class="form__label">Date</label>
+      <input type="text" name="text" id="inputDate" class="form__input" />
       <!-- add photo field-->
-      <label class="form__label form__label--url" for="inputImg" id="url"
-        >Photo</label
-      >
-      <input type="file" id="inputImg" class="form__input form__input--file" />
-      <!-- <input type="url" id="inputImg" class="form__input form__input--url" /> -->
+      <label class="form__label" for="inputImg">Photo</label>
+      <input type="file" id="inputImg" class="form__input" />
       <!-- radio buttons-->
       <div class="radio-buttons">
-        <input
-          id="rd-visited"
-          type="radio"
-          name="btn"
-          value="Visited"
-          label="Visited"
-        />
-        <label for="rd-visited" class="form__label--visited">Visited</label>
-        <input
-          id="rd-bucketlist"
-          type="radio"
-          name="btn"
-          value="Bucketlist"
-          label="Bucketlist"
-        />
-        <label for="rd-bucketlist" class="form__label--bucketlist"
-          >Bucketlist</label
-        >
+        <input id="rd-home" name="btn" type="radio" value="Home" />
+        <label for="rd-home" class="radio-label">Home</label>
+        <input id="rd-places" name="btn" type="radio" value="Places" />
+        <label for="rd-places" class="radio-label">Places</label>
       </div>
       <!-- Upload button -->
-      <button type="submit" id="upload" class="form-btn__submit">Upload</button>
+      <button type="submit" id="upload" class="form__submit">Upload</button>
       <!-- Close button -->
       <button
         type="button"
         id="close"
-        class="form-btn__close flaticon-001-cancel-3"
+        class="form__close flaticon-001-cancel-3"
       ></button>
     </form>
 
-    <!--M A I N - buttons-->
-    <div id="main-buttons" class="buttons container--main">
-      <span class="button buttons__view">My places:</span>
-      <button class="button button__visited" type="button">Visited</button>
-      <button class="button button__buckelist" type="button">Bucketlist</button>
-      <button class="button button__all" type="button">All</button>
-      <button
-        class="button button__add flaticon-016-add"
-        type="button"
-      ></button>
-    </div>
-
-    <!--M A I N - gallery-->
-    <main class="gallery container--main" id="gallery"></main>
+    <!-- gallery -->
+    <main class="gallery container" id="gallery"></main>
```

Comment:
The images loaded are heavy, they take time to load, and while that happens the layout breaks. [screenshot](https://user-images.githubusercontent.com/14869087/99878114-13913080-2bfb-11eb-8b34-bfcb700e3070.png)

This is a tricky problem, without perfect solutions taking into account your current knowledge level. My suggestion for you is to simplify. 

Set a predefined height for all images and use `object-fit`. That way the layout won't break and you won't need masonry. 

A mindset I like to follow is Small changes, Iterate a lot, Deploy often.

Focus on building the essential features and release the first version even if it's not your ideal final solution. Then, we can always iterate and improve things, one thing at a time.

Reply:
Are you sure it's the heavy images? Because I noticed there was a font library that was taking forever to load and once I removed it loading is back to normal, at least on my wifi.

Reply:
Locally, I added that font library (germano) again and the speed was still the same. But maybe that influenced it too.
However, I'm pretty sure the images are the main cause because a couple of them are quite heavy (+1Mb of size).

![image](https://user-images.githubusercontent.com/14869087/100021971-49404000-2dda-11eb-9ad1-753266e3869b.png)

For now, this isn't a major problem, both of us have nice internet speed. But once you start having a lot of images in your blog, you might want to do some performance adjustments. For example, uploading the same image in 2 different sizes using JS:
- small version eg 600px width
- original - full-resolution

In the gallery, we just load the small versions. Once you click on one of the photos, it loads the full resolution on a new page. I think it's a nice challenge for later ;)

Reply:
I'd love that! I resized all the images to under 300kb but it would just look terrible, that sounds like the perfect solution